stac_pydantic/version.py

```diff
@@ -1 +1 @@
-STAC_VERSION = "1.0.0-beta.2"
+STAC_VERSION = "1.0.0"
```

Comment:
This change, if omitted, allows tests to complete successfully.

Reply:
I'm rolling some example files that simply extend the previously working versions to use some 1.0.0 features. More canonical examples can be swapped in later if that's desirable

Reply:
And they are assembled here, temporarily: https://gist.github.com/moradology/977878279e1f36a6494877cdeaf18688

Reply:
I've gone ahead and folded the testing files within this repository